app/components/footer/secondary-section.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Grid } from '@trussworks/react-uswds';
+
+const SecondarySection = () => {
+  // TODO: Update the links to point to the correct pages
+  const secondaryNavItems = [
+    { label: 'Accessibility support', href: '/a' },
+    { label: 'No FEAR Act data', href: '/b' },
+    { label: 'Office of the Inspector General', href: '/c' },
+    { label: 'FOIA requests', href: '/d' },
+    { label: 'Performance reports', href: '/e' },
+    { label: 'Privacy policy', href: '/f' },
+  ];
```

Comment:
```suggestion
  const secondaryNavItems = [
    {
      label: 'Accessibility support',
      href: 'https://www.nasa.gov/accessibility/',
    },
    {
      label: 'No FEAR Act data',
      href: 'https://www.nasa.gov/no-fear-act/',
    },
    {
      label: 'Office of the Inspector General',
      href: 'http://oig.nasa.gov/',
    },
    {
      label: 'FOIA requests',
      href: 'https://www.nasa.gov/foia/',
    },
    {
      label: 'Performance reports',
      href: 'https://www.nasa.gov/organizations/budget-annual-reports/agency-financial-reports/',
    },
    {
      label: 'Privacy policy',
      href: 'https://www.nasa.gov/nasa-web-privacy-policy-and-important-notices/',
    },
  ];
```

@aboydnw there are some differences between the labels on Figma and Nasa.gov, so I took the closest match. For example, 'Performance reports' vs. 'Agency Financial Reports'. Could you please double-check my assumptions? 
Feel free to edit my comment, or commit the suggestion directly!